decidim/decidim
# Fix showing created meetings in 'my public profile'

#### :tophat: What? Why?
Created meetings are not displayed in My public profile.

We should have the same behavior like proposals

#### :pushpin: Related Issues

Related issue: https://github.com/decidim/decidim/issues/8518


#### Testing

1. Create a meeting (open or withdraw)
2. Check My my public profile - Timeline and Activity
3. Created meetings are displayed (withdraw and open)

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](https://github.com/decidim/decidim/blob/develop/CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
